tests/functional/firefox/os/version/test_2_0.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.os.version.v2_0 import FirefoxOSPage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_app_category_selector(base_url, selenium):
+    page = FirefoxOSPage(base_url, selenium).open()
+    app_groups = page.app_groups
+    for group in app_groups:
```

Comment:
We can just do `for group in page.app_groups` without the need for the `app_groups` variable.